incubator/spark-history-server/templates/deployment.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: spark-history-server
+  namespace: {{ .Release.Namespace }}
+---
```

Comment:
One file per resource, please.

Reply:
Ok. But is it ok to leave multiple resources in `rbac.yaml`. Each resource is a few lines of YAML and they are all RBAC-related.